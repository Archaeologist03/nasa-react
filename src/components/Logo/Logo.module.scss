.logoStyle {
  position: absolute;
  color: purple;
  font-size: 55px;
  z-index: 1;

  @media (min-width: 1024px) {
    left: 5%;
  }

  @media (max-width: 1024px) {
    right: 5%;
  }
}
