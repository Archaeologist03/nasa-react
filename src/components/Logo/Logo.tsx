import React from 'react';

import styles from './Logo.module.scss';
import nasaLogoImg from '../../assets/images/icons8-nasa.svg';

const Logo = () => {
  return (
    <div className={styles.logoStyle}>
      <img alt='' src={nasaLogoImg} />
    </div>
  );
};

export default Logo;
