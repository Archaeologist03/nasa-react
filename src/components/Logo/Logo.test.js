import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Logo from './Logo';

it('should render Logo component', () => {
  shallow(<Logo />);
});

it('should match Logo snapshot', () => {
  const wrapper = shallow(<Logo />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
