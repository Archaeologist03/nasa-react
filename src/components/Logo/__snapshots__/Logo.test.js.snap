// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Logo snapshot 1`] = `
<div
  className="logoStyle"
>
  <img
    alt=""
    src="icons8-nasa.svg"
  />
</div>
`;
